### DISCUSSION TITLE
用VPS远程编译AX6S时，卡在 Enables kernel network stack path for Shortcut  Forwarding Engine

### DISCUSSION THREAD

**Discussion by chika0801** [Boos4721/openwrt#122] [Q&A/答疑解惑] [answered]
</pre>
Support DMA zone (ZONE_DMA) [Y/n/?] y
Support DMA32 zone (ZONE_DMA32) [Y/n/?] y
ASN1 (ASN1) [M/y/?] m
Enables kernel network stack path for Shortcut  Forwarding Engine (SHORTCUT_FE) [N/y/?] (NEW) 

</pre>

每次都要手动输入 y 回车，才能继续。

但是用github在线编译，是直接完成的。

很奇怪。

我也没选SHORTCUT_FE，一样有这提示。

luci-app-turboacc (Flow Offload,BBR CCA)

**Comment by Boos4721** [comment, marked as answer]
KK ITS KERNEL CONFIG

**Reply by chika0801** [reply]
thanks .l love yuu.
